runtime/jcl/common/java_dyn_methodhandle.c

```diff
@@ -433,7 +433,7 @@ accessCheckMethodSignature(J9VMThread *currentThread, J9Method *method, j9object
 			endIndex = index;
 
 			/* If this entry is a class type, we need to do a classloader check on it */
-			if ('L' == lookupSigData[index]) {
+			if (IS_REF_OR_VAL_SIGNATURE(lookupSigData[sigOffset])) {
```

Comment:
I found the source of the build failure (I used `sigOffset` here by accident instead of `index`), and when I tested this code earlier I didn't test it on JDK8. But I'm not sure why this was only an error on JDK8 builds?

Reply:
The same error appears on line 465 as well.

Reply:
fixed in [e51ed56](https://github.com/eclipse-openj9/openj9/commit/e51ed56907924d04f98f890ee403b5dedea21856)